### DISCUSSION TITLE
"pre-update-check" that checks for corect fs permissions and a "post-update-check" that checks if all files have been installed correctly.

### DISCUSSION THREAD

**Discussion by daniel-lenz** [joomla/joomla-cms#47089] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
The problem is that it is often needed to re-install core files via manual downloaded archive until Joomla works as expected after an update.

### Describe the solution you'd like
Create a "pre-update-check" that checks for the richt filesystem permissions and a "post-update-check" that checks if all files have been installed correctly.

**Comment by kitepascal** [comment]
There is a check already at /administrator/index.php?option=com_admin&view=sysinfo

If 3.10.x Updates went fine there should not be any problems with permissions.

Bigger updates crash sometimes if the webhosting is low-pferformance garbage.

**Comment by daniel-lenz** [comment]
@kitepascal we host it on our own with really good performance server (16 GB Ram and 8 vCPUs).

**Comment by kitepascal** [comment]
That's the problem sometimes - Joomla is optimized and tested on modern, state-of-the-art, well configured webhostings.
But if you manage your own machine, it should be easier to analyse what went wrong.

**Comment by daniel-lenz** [comment]
these issues are occuring sporadically, often repairing the database or reinstalling core files from manually downloaded zip files a few times helps then. But if you don't have made this experience a couple of (100) times, how should one know it?
